test/tests/userobjects/layered_side_integral/tests

```diff
@@ -29,6 +29,13 @@
     requirement = "MOOSE shall allow taking averages of the flux of variables along a coordinate axis in layers on a boundary"
   [../]
 
+  [./flux_average_fv]
+    type = 'Exodiff'
+    input = 'layered_side_flux_average_fv.i'
+    exodiff = 'layered_side_flux_average_fv_out.e'
+    requirement = "MOOSE shall allow taking averages of the diffusive flux of finite volume variables along a coordinate axis in layers on a boundary"
```

Comment:
For SQA reporting I think these requirements are supposed to be complete sentences, so go ahead and make sure there are periods at the end.

Reply:
ok changed for the other tests in the file too, I m glad we will be back in compliance.